docs/features/write-mode-technical-badges.md

```
@@ -0,0 +1,238 @@
+
+### Icon Mapping
+
+| Badge Type | Icon | Lucide Name         |
+| ---------- | ---- | ------------------- |
+| Conditions | ❓   | `BadgeQuestionMark` |
+| Jump       | ↗    | `ArrowUpRight`      |
+| Visuals    | 🖼   | `Image`             |
+
+### Data Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)